app/screens/main/Main.js

```diff
@@ -99,15 +109,46 @@ class Main extends Component<Props, State> {
     );
   }
 
+  componentDidUpdate(prevProps, prevState) {
```

Comment:
what the scenario that can cause this?

Reply:
Refresh causes this.

Reply:
which refresh?

Reply:
Refreshing the app.

Reply:
we are using MemoryRouter now, every reload will cause landing at top route level no matter what caused it. Does this scenario you described still exist?

Reply:
I'll recheck and remove if so.